ui/apps/platform/cypress/integration/vulnerabilities/exceptionManagement/pendingRequestsTable.test.js

```diff
@@ -1,24 +1,17 @@
 import withAuth from '../../../helpers/basicAuth';
 import { hasFeatureFlag } from '../../../helpers/features';
+import { clearSearchFilters } from '../vulnerabilities.helpers';
 import {
     cancelAllCveExceptions,
     fillAndSubmitExceptionForm,
     selectSingleCveForException,
     verifyExceptionConfirmationDetails,
     verifySelectedCvesInModal,
     visitWorkloadCveOverview,
+    typeAndSelectCustomSearchFilterValue,
 } from '../workloadCves/WorkloadCves.helpers';
-import { selectors as workloadCVEsSelectors } from '../workloadCves/WorkloadCves.selectors';
 import { visitExceptionManagement } from './ExceptionManagement.helpers';
-
-function clearWorkloadCVEsFilters() {
```

Comment:
Moved this into it's own file above (vulnerabilities.helpers.js). Also, renamed it to be clearer and more inclusive